eva_submission/eload_utils.py

```diff
@@ -68,7 +68,9 @@ def get_metadata_conn():
 
 def get_mongo_creds():
     properties = get_properties_from_xml_file(cfg['maven']['environment'], cfg['maven']['settings_file'])
-    mongo_host = split_hosts(properties['eva.mongo.host'])[0][0]
+    # Use the primary mongo host from configuration:
+    # https://github.com/EBIvariation/configuration/blob/master/eva-maven-settings.xml#L111
+    mongo_host = split_hosts(properties['eva.mongo.host'])[1][0]
```

Comment:
This is a temporary fix until we can determine if the accessioning/variant pipelines can support multiple hosts. Can we put in a TODO so that we can come back and revert this change later?

Reply:
I've added TODOs for both changes in this PR as they're both temporary changes.